src/java.desktop/share/classes/javax/swing/JFormattedTextField.java

```diff
@@ -1086,25 +1086,27 @@ protected void setEditValid(boolean valid) {
         }
 
         /**
-         * @implNote Default implementation returns null.
          * Subclass must override if you wish to provide a
          * <code>DocumentFilter</code> to restrict what can be input.
          * <code>install</code> will install the returned value onto
          * the <code>JFormattedTextField</code>.
          *
+         * @implNote The default implementation returns <code>null</code>.
+         *
          * @return DocumentFilter to restrict edits
          */
         protected DocumentFilter getDocumentFilter() {
             return null;
         }
 
         /**
-         * @implNote Default implementation returns null.
          * Subclass must override if you wish to provide a filter to restrict
          * where the user can navigate to.
          * <code>install</code> will install the returned value onto
          * the <code>JFormattedTextField</code>.
          *
+         * @implNote The default implementation returns <code>null</code>
```

Comment:
```suggestion
         * @implNote The default implementation returns <code>null</code>.
```
Full stop for consistency.